app/vmselect/netstorage/netstorage.go

```diff
@@ -1705,7 +1705,10 @@ func (sn *storageNode) execOnConn(qt *querytracer.Tracer, funcName string, f fun
 	if err := writeBool(bc, traceEnabled); err != nil {
 		// Close the connection instead of returning it to the pool,
 		// since it may be broken.
-		_ = bc.Close()
+		if er := bc.Close(); er != nil {
```

Comment:
I don't think the `er` is needed to be logged, since the root cause error is in the `err`. We just need to close the `bc` here.